scripts/player_audio_manager.gd

```diff
@@ -0,0 +1,38 @@
+extends Node
+
+@onready var landing_sfx: AudioStreamPlayer2D = $"Landing SFX"
+@onready var footstep_sfx: AudioStreamPlayer2D = $"Footstep SFX"
+@onready var jumping_sfx: AudioStreamPlayer2D = $"Jumping SFX"
+@onready var moth_jump_sfx: AudioStreamPlayer2D = $"Moth Jump SFX"
```

Comment:
I think this is fine but I think it is better to usually export it so there are no dependencies on structure / name. If you move the sfx somewhere else or name it differently it will cause an issue in code. If you do that with an export it will keep the reference still. If you still want to use the @onready annotation you can still put it in _ready() but I dont think it is needed.